README.md

```diff
@@ -60,6 +60,16 @@ await ZendeskMessaging.logoutUser();
 await ZendeskMessaging.loginUserCallbacks(jwt: "YOUR_JWT", onSuccess: (id, externalId) => ..., onFailure: () => ...;
 await ZendeskMessaging.logoutUserCallbacks(onSuccess: () => ..., onFailure: () => ...);
 ```
+### Retrieve the unread message count (optional)
+
+There's must be a logged user to allow the recovery of the unread message count!
+
+```dart
+// Retrieve the unread message count
+final int count = await await ZendeskMessaging.messageCount();
```

Comment:
remove one await, and use getUnreadMessageCount()

Reply:
Fixed in 20b802ed8f51d865d4df359d352885757e5c5a9c